variants/templates/variants/filter_result/row.html

```diff
@@ -572,6 +572,17 @@
             </span>
           {% endif %}
 
+          {% linkout_available_for "franklin" entry.release as franklin_available %}
+          {% if franklin_available %}
+            <a class="dropdown-item" href="{% linkout_base_url "franklin" entry.release %}{{ entry.chromosome }}-{{ entry.start }}-{{ entry.reference }}-{{ entry.alternative }}" target="_blank">
```

Comment:
`-{{ entry.alternative }}` should be suffixed by `-hg38` for GRCh38 or no suffix for hg19/GRCh37